src/content/app/genome-browser/components/zmenu/zmenus/VariantZmenu.tsx

```diff
@@ -82,4 +90,39 @@ const VariantZmenu = (props: Props) => {
   );
 };
 
+/**
+ * Consider that the genome browser shows the detailed zoomed-in view for variants
+ * at scales 2**1 to 2**6 base pairs. 2**6 is 64; and when the genome browser zooms out to this point,
+ * it switches to the zoomed-out view.
+ */
+const MAX_NUCLEOTIDES_PER_VIEWPORT = 64 - 2;
```

Comment:
Can't the value of this just be `62`?

Reply:
I was trying to make it explicit that 64 is 2**6. I should probably make it even more explicit still. Like:


```js
const VARIANT_DETAILS_TRACK_NUCLEOTIDES_LIMIT = 2 ** 6 - 1; // 63 nucleotides
const MAX_NUCLEOTIDES_PER_VIEWPORT = VARIANT_DETAILS_TRACK_NUCLEOTIDES_LIMIT - 1; // to make it an even number
```

Reply:
This would actually be easier to understand